client/app/components/packages/landuse-form/proposed-action-editor.hbs

```diff
@@ -15,9 +20,163 @@
       </span>
     </Ui::Legend>
 
+    <Ui::Question
+      class="no-margin-top"
+      @required={{true}}
+      data-test-dcpapplicantispublicagencyactions-radio-group
+      as |dcpApplicantispublicagencyactionsQ|
+    >
+      <dcpApplicantispublicagencyactionsQ.Legend>
+        Is this an action where the applicant is solely a public agency?
+      </dcpApplicantispublicagencyactionsQ.Legend>
+
+      <landuseActionForm.Field
+        @attribute="dcpApplicantispublicagencyactions"
```

Comment:
weird, how come I don't see this field in AB#12683?

Reply:
I think it was included in other PRs so they didn't include it in this task